nntrainer/layers/cl_layers/fc_layer_cl.h

```diff
@@ -101,12 +101,19 @@ class FullyConnectedLayerCl : public LayerImpl {
    */
   void setProperty(const std::vector<std::string> &values) override;
 
+  static bool registerClKernels() { return true; };
+
   inline static const std::string type = "fully_connected";
 
 private:
   std::tuple<props::Unit>
     fc_props; /**< fc layer properties : unit - number of output neurons */
   std::array<unsigned int, 2> weight_idx; /**< indices of the weights */
+
+  const static int num_layer_kernels = 0; /** < number of layer kernels */
```

Comment:
it looks like num_layer_kernels is unused. what is it for?

Reply:
oh it is the previous version lol! Thank you for your comment. I tried to add kerenel registration check through num_layer_kernels == len() thing... but I decided to remove it. I'm gonna update the code. Thank you.